app/helpers/conditions-for-contrib-removal.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+/**
+ * conditionsForContribRemoval helper - used to determine if the removing a particular
+ * contributor will still satisfy two conditions 1) @ least one registered admin 2) @ least one
+ * bibliographic contributor
+ *
+ */
+export function conditionsForContribRemoval(params/*, hash*/) {
+    var contributorToRemove = params[0];
+    var contributors = params[1];
+    if (contributors) {
+        var minRegisteredAdmins = false;
+        var minBibliographic = false;
+        contributors.forEach(function(contributor) {
+            if (contributor.id !== contributorToRemove.id) {
+                if (contributor.get('permission') === 'admin' && contributor.get('unregisteredContributor') === null) {
+                    minRegisteredAdmins = true;
+                }
+                if (contributor.get('bibliographic')) {
+                    minBibliographic = true;
+                }
+            }
+        });
+        return minRegisteredAdmins && minBibliographic;
+    } else {
+        return params;
```

Comment:
Hmm. The downside is that even an empty array evaluates as truthy- in the absence of data, the page is still making decisions based on this result. That may not be what was intended.

Is there no data at all, or just no useful default?

`if([]){console.log('whee')}`